x-pack/test/functional/apps/security/users.ts

```diff
@@ -37,7 +41,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       const users = keyBy(await PageObjects.security.getElasticsearchUsers(), 'username');
       log.info('actualUsers = %j', users);
       log.info('config = %j', config.get('servers.elasticsearch.hostname'));
-      if (config.get('servers.elasticsearch.hostname') === 'localhost') {
+
+      // In Cloud default users are defined in file realm, such users aren't exposed through the Users API.
+      if (isCloudEnvironment()) {
+        expect(Object.keys(users)).to.eql(['test_user']);
```

Comment:
Can this change? Is there a skipCloud tag we could use similar to skipFirefox?

Reply:
> Is there a skipCloud tag we could use similar to skipFirefox?

There is `skipCloud`, but it'd still be beneficial to test the Cloud default users (or lack thereof). I think what we need is `onlyCloud` so that we can have a separate test suite for the Cloud, but afaik we don't have this yet.